content/en/tutorials/lotus-miner/msig-as-owner.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Multisignature address as owner address"
+description: "This is a step-by-step guide on how to set a multisignature address as the owner address of a storage provider"
+lead: "This is a step-by-step guide on how to set a multisignature address as the owner address of a storage provider. This tutorial is for experienced Lotus users. Setting a multisignature address as an owner address has some serious UX drawbacks but adds additional layers of security."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-providers"
+weight: 140
+toc: true
+---
+
+## Prerequisites
+
+To change the owner address of the storage provider to a multisignature address you will need to have:
+
+- A [multisignature address]({{< relref "../../../lotus/manage/multisig/" >}}).
+- [Lotus-shed]({{< relref "../../../kb/lotus-shed-not-installed/" >}}) installed.
+
+### Advantages
+- Using a multisignature address adds a lot of security to the owner address as multiple signers need to sign a message for it to be executed.
+
+### Disadvantages
+
+- Some commands like `lotus-miner actor withdraw` currently do not support multisignature addresses, and you will need to use the `lotus-shed` tool to withdraw from the available actor balance.
```

Comment:
```suggestion
### Advantages

Using a multisignature address adds a lot of security to the owner address as multiple signers need to sign a message for it to be executed.

### Disadvantages

Some commands like `lotus-miner actor withdraw` currently do not support multisignature addresses, and you will need to use the `lotus-shed` tool to withdraw from the available actor balance.
```

If each section only has 1 bullet-point, it's better to just leave them as paragraphs.